### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by ccmdrd** [HelloNYAC/JobVacancyRate-LMINews#2]
## TL;DR
I think overall this is good and you're able to show off your understanding of the data and get some practice writing html and css, and using the statcan API, etc.

I do have some comments in terms of the components you used and some suggestions for what to use if you were to build something like this again in the future to adhere to some web standards and accessibility. I understand this wasn't really a requirement because this is just a way for you to demonstrate what you've learned during your co-op term, so let this just be an opportunity to learn because we didn't really go through this together as you were building it

## WET Styleguide
https://wet-boew.github.io/wet-boew-styleguide/index-en.html
It's good to get in the habit of using the styleguide as a reference when you're building things for the web, especially in the government of Canada. I usually check here often when I have an idea for a component on a page to see if there is already an acceptable way to do it to maintain a common look, feel, and interaction across sites. If what I'm looking for doesn't exist in the styleguide, then I would begin looking into making it myself.

I'll be providing some alternative components found in the styleguide that probably would have saved you some trouble

### toggle
https://wet-boew.github.io/v4.0-ci/demos/toggle/toggle-en.html
`.home-quotes` would have been better suited as toggles, which are able to hide and reveal details. They are also keyboard accessible and don't need extra code to work. The toggle will also have focus states that make it clear to keyboard users if they are about to select a toggle to reveal/hide details

### lists
https://wet-boew.github.io/wet-boew-styleguide/design/lists-en.html
`.manual_details`  would have been better as `<li>` list items inside of a `<ul>`. Semantically, the items in a list should use the list element to inform the  user that what they are reading is a list. You also don't need to add any code to get the bullets, and it is more manageable to have a custom bullet if you wanted to keep the arrows

### tables
https://wet-boew.github.io/wet-boew-styleguide/design/tables-en.html
The bootstrap table is good but I'm sure you can notice a jarring shift in style

### Social media icons
I suggest removing the social media icons that don't lead anywhere so that they don't distract users nor interfere with keyboard users since they don't lead to anything

### Search field
Same suggestion as above

### Navigation bar
I'm curious if you added the `tab-index="-1"` to some of the items in the navigation, because this prevents users from selecting them using the keyboard.

### Focus and hover states
https://wet-boew.github.io/wet-boew-styleguide/accessibility/requirements/ui-controls-en.html?wbdisable=true
A lot of the interactive items on the page are missing focus states which let the user know where they are if they are a keyboard user. 

This is especially helpful for a page with a form like your Job Vacancy page - there are even some parts I can't interact with using the keyboard

A very simple way to add a focus state is by adding this rule in your css

```
/* This will draw a black box around a component if it has keyboard focus */
[some selector]:focus {
  outline: 4px solid black;
}
```

### Economic Regions
I remember you had a previous version with Economic Region data. This would be the most useful information to display because the provincial data may not be very representative of the reality in a smaller ER. 

### Compiling information
Because the filters for each of your three actionable forms are very similar, it would be more useful if there was one form, that provides the, Job Vacancy Rates, Job Vacancies and LMI News on the results page

In the same way that displaying just one value isn't very useful, and providing the same stat but at a higher level for comparison, providing the rate, with the count gives an extra level of context that's missing when you separate them

### Content Prioritization
Specifically at the results page it takes a second to remind myself what exactly I'm looking at. I would make it very clear what a specific value is (I understand it takes an extra step to change the table headers to be more specific, but even making it clear in the title of the table helps a lot)

### Questionable stats
I'm not sure if averaging the job vacancy rates across all industries is very helpful nor comparable
also, not sure if taking the average job vacancies is very helpful
it may be better to discuss this on a call because it may be complicated to explain
